app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -1,6 +1,12 @@
 # Copyright © Magento, Inc. All rights reserved.
 # See COPYING.txt for license details.
 
+type StoreConfig @doc(description: "The type contains information about a store config") {
+    required_character_classes_number : String @doc(description: "Number of Required Character Classes")
+    minimum_password_length : String @doc(description: "Minimum Password Length")
+    autocomplete_on_storefront : Boolean @doc(description: "Enable Autocomplete on login or forgot password forms")
```

Comment:
I see it's `boolean` here but sting in `di`. Could you please add test cases for `StoreConfig` as well.

Reply:
**Done**